src/System.Net.HttpListener/src/System/Net/HttpListener.Windows.cs

```diff
@@ -102,104 +71,69 @@ internal SafeHandle RequestQueueHandle
             }
         }
 
-        public AuthenticationSchemeSelector AuthenticationSchemeSelectorDelegate
+        private void ValidateV2Property()
```

Comment:
There is no Pal layer on Unix, the files are split with os-specific differences in HttpListener.windows.cs and HttpListener.unix.cs. Only Pal is not there in filename. The common code is not diverged here, all common code was initially in HttpListener.Windows.cs, since there was no Unix implementation. This change merely collects all common code to HttpListener.cs